### PR TITLE
feat: working version of android attestation poc

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Tried to keep it simple. There are couple things we can do to increase the assurance. 
1) Cache our nonce's with their associated connection id and verify that the original nonce matches the nonce returned with the integrity verdict (see the sample verdict JSON)
2) Include the check for the `appRecognitionVerdict` being `PLAY_RECOGNIZED` but that breaks testing it in development

### CODE REVIEW

File: src/__pycache__/apple.cpython-311.pyc
Comment (jleach) [inline src/__pycache__/apple.cpython-311.pyc:1]: Can we remove?

File: src/__pycache__/traction.cpython-311.pyc
Comment (jleach) [inline src/__pycache__/traction.cpython-311.pyc:1]: Can we remove?

**Review by jleach — CHANGES_REQUESTED** [review]

File: src/goog.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
...M_Fp66gPwYphTsJwY1eF-vYnKGTnHtxxIIuriNAHBxPBtyzauEVC4eNkz-wQwoRUx7WjpMpIqDpLRpIDFJ4D0cbbsEiqoSMHY3ik5xIWWuZfj84tL8AJ_Eq28nO9BlAKZoIK-mMbiyQzdyyVrSTi2B9c3TGyEQ40M6He_odCxfoeDlIic-yjyqMYyEFyCRy_WOAHcDf2FGe7NNuGH2HTsmYCC337k3l99CO83aWEjfGUlxtJkZSPtsF43x9jMlcJekN7mF0p86U1bZ2M_cj-wKyvib2f-giA74xaTOP5r0dNivURwAX8UebwRRL4srnwqs3UfAwsBiDXa5N0b8xMMMyhBUdelVDqr8x8FTvSUmNa82PrHN3VAde_MaimEzRcbm7d885Q_iOin6AfR4cEw_hhaBRAUv_OK71XTHBPGXeSy-UDaNZMY3XonixANCwUBk-d1uTzuGg98ePwcgx03wY9XZc9vKi5mbvSR2Za9GXRcevAJQ5GBPhGPMt7FCN7vYhcPO2gUdO2P4kOtdJs1R4AK3FRev3JFT9Ruv7W1o1tZNzoIF5ClILk07bVv9gEX7joy7ipGLMJa03vDzG7r_ynCwdrUAystRYkvdMotOlUqAHDRn67dhXi6prToykPq5LiFHg4qt1F0DWUiOsYo81-hTeutwgwAvDPgGwYZDIuK5oEkAuEzoyJ7Ek3pmLQDXxmUg1tcn9xmRniLcOd5ZaH03pU7N1nz9xOuI0NyYOnzYuqr0bVhnlatadytAz5Pj8Nh5vg0lOMGz8nlgCQxv0Aku19QKrCk_DHP1QRkGtdISoPPHUakPaSe5iPq1sH5TrOqZOGrWK-shyqExxpjq-rCHaUv10SYB2XM8cg2wD1cuwfNeFJ8g6NpHc34r8NTVOgD_dOGnI_WKxmomqD7jltALfm-u7NnUGMHQnRNIcQl2NZp8KuQI1jbjHpA3Vz-mBwIoYFbNexeKZxG9fN6PoDb2XFPAQXAx1yXRz3KiCHsbOqhgy2gaXC2opuxKKotjTNFiS0U-SKr8eNZxBjap_PG11F--pqvNgM9LA4wMJrN0Fl3VvrIVKEEkl3CLP8Dq3XBAdno1RuPB1H32CpDU1Rni8pkKKuarTedyOLFEZRKgTbbdEcsdIkPF1yHQqbnh857EAOIQE.n0zsMOeoaBpqK9sleN6CJw" # noqa: E501
```
Comment (jleach) [inline src/goog.py:58]: Can we remove the comment?

**Review by jleach — CHANGES_REQUESTED** [review]

File: src/__pycache__/apple.cpython-311.pyc
Comment (bryce-mcmath) [reply src/__pycache__/apple.cpython-311.pyc:1]: this file is being deleted

File: src/__pycache__/traction.cpython-311.pyc
Comment (bryce-mcmath) [reply src/__pycache__/traction.cpython-311.pyc:1]: same as above

File: src/goog.py (hunk 1 above)
Comment (bryce-mcmath) [reply src/goog.py:58]: will do

**Pull request merged**